client/app/scripts/liveblog-syndication/views/consumer-list-item.html

```diff
@@ -0,0 +1,34 @@
+<li ng-repeat="consumer in consumers" ng-click="select(consumer)" ng-class="{selected: consumer === selected, 'inactive' : !consumer.is_enabled || !consumer.is_active}">
+    <figure class="avatar avatar-with-info" ng-class="{'consumer-logged': isLoggedIn(consumer)}">
+        <div class="logged-info"></div>
+        <div class="consumer-image">
+            <img sd-consumer-avatar data-consumer="consumer">
+        </div>
+    </figure>
+
+    <i class="admin-label icon-settings" ng-if="consumer.consumer_type == 'administrator'" tooltip="{{ :: 'admin'|translate}}" tooltip-placement="top"></i>
```

Comment:
Again, this is legacy from user related code, I think. The consumer cannot be of type administrator. Shouldn't you be checking the logged in user here, for consumer edit rights?